google-cloud-spanner/src/main/java/com/google/cloud/spanner/InstanceInfo.java

```diff
@@ -68,8 +77,22 @@ static FieldMask toFieldMask(InstanceField... fields) {
 
     public abstract Builder setDisplayName(String displayName);
 
+    /**
+     * Sets the number of nodes for the instance. Only one of processing units or node count must be
```

Comment:
Must or can? If must, then "Exactly one of"

Reply:
Must, changed the wording to "Exactly one of"